index.html

```diff
@@ -1,13 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-  <link rel="stylesheet" href="styles/index.css">
-</head>
-<body>
-  <h1>Quiz-App</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App start page</title>
+    <link rel="stylesheet" href="styles/index.css" />
+  </head>
+  <body class="app-grid">
+    <header id="#starthome" class="top">
+      <h1 class="top quiz-app-heading">Quiz-App</h1>
+    </header>
+
+    <main class="app-grid__scrollarea h-100vh">
+      <section class="quiz-app-card">
+        <article class="quiz-app-card-question">
+          <h2 class="quiz-app-card__heading">Question:</h2>
+          <p class="quiz-app-card-question--text"></p>
+        </article>
+        <figure class="bookmark-container">
+          <img src="" alt="" width="" height="" class="bookmark--on" />
+        </figure>
+        <button class="quiz-app-card-answer--show"></button>
+        <button class="quiz-app-card-answer--hide"></button>
+        <p class="quiz-app-card-question--answer"></p>
+        <div class="quiz-app-card-advise-tag__container">
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+        </div>
+      </section>
+    </main>
```

Comment:
it´s nice structured 👍  
I´m not sure if you really need the `section` as a container...because `main` is already one!?
for the class naming...if you want to stick to BEM it´s better if you for eg rename first <section class="cards"> this would be shorter instead of quiz-app-card  😄 then you can use cards for the following elements as starter eg `cards__body `and/or `cards__title` and/or `cards__question`

Reply:
the section i thought for one single card - when this goes okay i want to use it also at the bookmarks page

you are right - but as i wanted to go with bem, i accepted the long names - i am also curious about the judging of the coaches 😃